packages/docusaurus-plugin-content-blog/src/blogUtils.ts

```diff
@@ -73,7 +75,7 @@ export function paginateBlogPosts({
 
   function permalink(page: number) {
     return page > 0
-      ? normalizeUrl([basePageUrl, `page/${page + 1}`])
+      ? normalizeUrl([basePageUrl, `${pageBasePath}/${page + 1}`])
```

Comment:
It's not your fault, but I'm not a fan of the `**DO NOT** include a trailing slash` (for both this feature and the `tagsBasePath` too)

Maybe we could get rid of it if we just did this instead?

```js
normalizeUrl([basePageUrl, pageBasePath, `${page + 1}`])
```

This way we don't care if the user provides a slash or not.

I'd be happy to do some extra cleanup here

Reply:
Reworked using your suggestion.